src/services/import-manifest-service.ts

```diff
@@ -7,21 +7,37 @@ import { PromptService } from 'services/prompt-service';
 import { ImportCommandTasksContext } from 'types/commands/manifest-import';
 import { AppId, AppVersionId } from 'types/general';
 import { HttpMethodTypes } from 'types/services/api-service';
+import { loadFile, saveToFile } from 'utils/file-system';
+import { processTemplate } from 'utils/templating';
 import { TIME_IN_MILLISECONDS } from 'utils/time-utils';
 import { appsUrlBuilder } from 'utils/urls-builder';
 
 export const uploadManifestTsk = async (
   ctx: ImportCommandTasksContext,
   task: ListrTaskWrapper<ImportCommandTasksContext, any>,
 ) => {
+  task.output = `Processing manifest templating`;
+  const processedManifest = await processManifestTemplate(ctx.manifestFilePath, ctx.templateVars);
```

Comment:
wrap the whole thing with try catch, in case the format is not JSON or for other reason